packages/react-table/src/components/TableComposable/examples/ComposableTableActions.tsx

```diff
@@ -25,26 +20,67 @@ interface Repository {
   prs: string;
   workspaces: string;
   lastCommit: string;
+  singleAction: React.ReactNode;
 }
 
 type ExampleType = 'defaultToggle' | 'customToggle';
 
 export const ComposableTableActions: React.FunctionComponent = () => {
   // In real usage, this data would come from some external source like an API via props.
   const repositories: Repository[] = [
-    { name: 'one', branches: 'two', prs: 'a', workspaces: 'four', lastCommit: 'five' },
-    { name: 'a', branches: 'two', prs: 'k', workspaces: 'four', lastCommit: 'five' },
-    { name: 'p', branches: 'two', prs: 'b', workspaces: 'four', lastCommit: 'five' },
-    { name: '4', branches: '2', prs: 'b', workspaces: 'four', lastCommit: 'five' },
-    { name: '5', branches: '2', prs: 'b', workspaces: 'four', lastCommit: 'five' }
+    {
+      name: 'one',
+      branches: 'two',
+      prs: 'a',
+      workspaces: 'four',
+      lastCommit: 'five',
+      singleAction: <Button variant="secondary">Start</Button>
```

Comment:
Same here, we should render the Button as part of mapping over this array to convert it to JSX, not as a property of the array itself IMO.